CMakeLists.txt

```diff
@@ -128,6 +128,10 @@ if(WITH_PIC OR ENABLE_SHARED)
         ${CMAKE_CURRENT_BINARY_DIR}/config
         ${CMAKE_CURRENT_LIST_DIR}/src/include
     )
+    # Append any provided C flags.
+    if(COMPILER_DIAGNOSTIC_C_FLAGS)
```

Comment:
Should we do the same for `COMPILER_DIAGNOSTIC_CXX_FLAGS`?

Reply:
Good question, that's not necessary, We don't have any CXX sources in the WiredTiger library.